syntax = "proto3";

option java_multiple_files = true;
option java_package = "io.grpc.examples.removerruido";
option java_outer_classname = "RemoverRuidoProto";
option objc_class_prefix = "HLW";

package removerruido;


service ReduceNoise {
  rpc NoiseReducer (ReduceRequest) returns (ReduceRequest) {}
}

// O pedido com o audio e o frequencia dele.
message ReduceRequest {
  bytes audio = 1;
  int32 rate = 2;
}

// A resposta com o audio tendo menos ru√≠do.
message ReduceReply {
  bytes audio = 1;
}
